lua/starfall/libs_sh/entities.lua

```diff
@@ -1197,4 +1197,72 @@ function ents_methods:entOwner()
 	return owrap(ent:GetOwner())
 end
 
+--- Gets the bounds (min and max corners) of a hit box.
+-- @shared
+-- @param hitbox The number of the hitbox.
+-- @param group The number of the hitbox group, 0 in most cases.
+-- @return Hitbox mins vector.
+-- @return Hitbox maxs vector.
+function ents_methods:getHitBoxBounds(hitbox, group)
+	checkluatype(hitbox, TYPE_NUMBER)
+	checkluatype(group, TYPE_NUMBER)
+	local ent = getent(self)
+	local mins, maxs = ent:GetHitBoxBounds(hitbox, group)
+	if mins and maxs then
+		return vwrap(mins), vwrap(maxs)
+	end
+end
+
+--- Gets number of hitboxes in a group.
+-- @shared
+-- @param group The number of the hitbox group.
+-- @return Number of hitboxes
+function ents_methods:getHitBoxCount(group)
+	checkluatype(group, TYPE_NUMBER)
+	local ent = getent(self)
+	return ent:GetHitBoxCount(group)
+end
+
+--- Gets the bone the given hitbox is attached to.
+-- @shared
+-- @param hitbox The number of the hitbox.
+-- @param group The number of the hitbox group, 0 in most cases.
+-- @return Bone ID
+function ents_methods:getHitBoxBone(hitbox, group)
+	checkluatype(hitbox, TYPE_NUMBER)
+	checkluatype(group, TYPE_NUMBER)	
+	local ent = getent(self)
+	return ent:GetHitBoxBone(hitbox, group)
+end
+
+--- Returns entity's current hit box set.
+-- @shared
+-- @return Hitbox set number, nil if entity has no hitboxes.
+-- @return Hitbox set name, nil if entity has no hitboxes.
+function ents_methods:getHitBoxSet()	
+	local ent = getent(self)
+	local num, str = ent:GetHitboxSet()
```

Comment:
My only suggestion is you can return here instead of storing as locals first

Reply:
return getent(self):GetHitboxSet()

Reply:
Done!